auto_submit/lib/validations/ci_successful.dart

```diff
@@ -152,4 +172,16 @@ class CiSuccessful extends Validation {
 
     return allSuccess;
   }
+
+  List<String> _checkRunListToStringList(List<github.CheckRun> checkRuns) {
+    List<String> checkRunStrings = [];
+    for (int i = 0; i < checkRuns.length; i++) {
+      checkRunStrings.add(_checkRunToString(checkRuns[i]));
+    }
+    return checkRunStrings;
+  }
+
+  String _checkRunToString(github.CheckRun checkRun) {
```

Comment:
You can use `checkrun.toJson()`. It'll convert nicely in the logs to the json format

Reply:
Good point. Let me do that.

Reply:
Nvm I see. Let me do this.